Sources/GRPCCore/GRPCServer.swift

```diff
@@ -154,13 +154,13 @@ public struct GRPCServer: Sendable {
   ///
   /// This function returns when the configured transport has stopped listening and all requests have been
   /// handled. You can signal to the transport that it should stop listening by calling
-  /// ``stopListening()``. The server will continue to process existing requests.
+  /// ``beginGracefulShutdown()``. The server will continue to process existing requests.
   ///
   /// To stop the server more abruptly you can cancel the task that this function is running in.
   ///
   /// - Note: You can only call this function once, repeated calls will result in a
   ///   ``RuntimeError`` being thrown.
-  public func run() async throws {
+  public func serve() async throws {
```

Comment:
I am a bit wary of renaming this from `run` to `serve`. While I understand what you are getting at. We do have many libraries that now adopted the `run` method name to align with what `Lifecycle` expects. If you rename this here and conform to `Service` in the future your type will have two methods that do the same.

Reply:
`run` is less clear and is worse for users which don't depend on lifecycle. Users would have to provide shims to use `Lifecycle` anyway (to avoid retroactive conformance). If we provide conformance in the future (e.g. via package traits) then having a `run()` method calling through to `serve()` isn't an issue.

Reply:
From the perspective of a user using `Lifecycle` the names of these methods don't matter at all because they won't be called directly by the user.

From the perspective of a user _not_ using `Lifecycle`, these methods matter a lot: they _will_ be called by the user so they should be clear.

Reply:
I still remain unconvinced. `serve` and `run` are pretty close in my mind. Even leaving `Lifecycle` aside just the proliferation of `run` methods across the ecosystem is IMO enough to justify aligning. It becomes very natural that you call `run` on your server/clients when you don't use something like `Lifecycle`.

Reply:
Noted. I'm pretty strongly against aligning for the sake of aligning on a generic name: I think we should pick names that express their intent and provide the most clarity, the APIs should be approachable for all users including those new to Swift and SoS!